alchemist/alchemist-euclidean-geometry/src/test/kotlin/it/unibo/alchemist/test/TestEuclidean2DShapeFactory.kt

```diff
@@ -11,6 +11,15 @@ import it.unibo.alchemist.model.interfaces.geometry.euclidean2d.Euclidean2DShape
 import it.unibo.alchemist.model.interfaces.geometry.euclidean2d.Euclidean2DShapeFactory
 import it.unibo.alchemist.model.interfaces.geometry.euclidean2d.Euclidean2DTransformation
 
+internal fun Euclidean2DShapeFactory.oneOfEachWithSize(size: Double) = mapOf(
```

Comment:
Private or needs refactoring.

Reply:
Moved to the companion object of `TestEuclidean2DShapeFactory`.